code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -1505,7 +1505,7 @@
 
 			if ("pee", "piss", "urinate")
 				if (src.emote_check(voluntary))
-					if (sims)
+					if (sims?.getValue("Bladder") != null)
```

Comment:
Could also do
```
var/bladder = sims?.getValue("Bladder")
if (bladder)
...
```
to prevent calling `getValue()` twice. I don't actually know which one is better though, so it's fine if you stick with your current code.

Reply:
fixed in latest commit